purchase_request/wizard/purchase_request_line_make_purchase_order.py

```diff
@@ -348,8 +348,13 @@ class PurchaseRequestLineMakePurchaseOrderItem(models.TransientModel):
         string="Quantity to purchase", digits="Product Unit of Measure"
     )
     product_uom_id = fields.Many2one(
-        comodel_name="uom.uom", string="UoM", required=True
+        comodel_name="uom.uom",
+        string="UoM",
+        related="product_id.uom_po_id",
```

Comment:
A default value is may be better. Otherwise user can't decide which UoM to use.

Reply:
Just did a small test in runboat. The product_uom_id is always reset to product_id.uom_po_id